Description: add GCC hardening.
Author: Luis Paulo Linares <lpfllinares@gmail.com>
Last-Update: 2020-06-01
Index: odt2txt-0.5/Makefile
===================================================================
--- odt2txt-0.5.orig/Makefile
+++ odt2txt-0.5/Makefile
@@ -7,7 +7,7 @@ CFLAGS = -O0 -g -Wextra -DMEMDEBUG -DSTR
 #LDFLAGS = -lefence
 LDFLAGS += -g
 else
-CFLAGS = -O2
+CFLAGS += -O2
 endif
 
 ifdef NO_ICONV
